Description: Fix Maven metadata
 The POM files were including an incorrect version in all artifact's
 specifications.
Author: Miguel Landaeta <miguel@miguel.cc>
Forwarded: not-needed
Last-Update: 2010-06-23

diff --git a/anttask/pom.xml b/anttask/pom.xml
index decda0d..5e278e3 100644
--- a/anttask/pom.xml
+++ b/anttask/pom.xml
@@ -6,6 +6,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>org.codehaus.castor</groupId>
 	<artifactId>castor-anttasks</artifactId>
+	<version>1.3.2</version>
 
     <parent>
         <groupId>org.codehaus.castor</groupId>
diff --git a/codegen/pom.xml b/codegen/pom.xml
index 3253f21..71b58db 100644
--- a/codegen/pom.xml
+++ b/codegen/pom.xml
@@ -6,6 +6,7 @@ http://maven.apache.org/maven-v4_0_0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>org.codehaus.castor</groupId>
 	<artifactId>castor-codegen</artifactId>
+	<version>1.3.2</version>
 
 	<parent>
 		<groupId>org.codehaus.castor</groupId>
diff --git a/core/pom.xml b/core/pom.xml
index e06ca21..f8934ec 100644
--- a/core/pom.xml
+++ b/core/pom.xml
@@ -6,6 +6,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.codehaus.castor</groupId>
     <artifactId>castor-core</artifactId>
+    <version>1.3.2</version>
 
     <parent>
         <groupId>org.codehaus.castor</groupId>
diff --git a/cpa/pom.xml b/cpa/pom.xml
index 90364eb..19e2ab5 100644
--- a/cpa/pom.xml
+++ b/cpa/pom.xml
@@ -6,6 +6,7 @@ http://maven.apache.org/maven-v4_0_0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>org.codehaus.castor</groupId>
 	<artifactId>castor-jdo</artifactId>
+	<version>1.3.2</version>
 
     <parent>
         <groupId>org.codehaus.castor</groupId>
diff --git a/ddlgen/pom.xml b/ddlgen/pom.xml
index b19418f..3bc7f27 100644
--- a/ddlgen/pom.xml
+++ b/ddlgen/pom.xml
@@ -6,6 +6,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>org.codehaus.castor</groupId>
 	<artifactId>castor-ddlgen</artifactId>
+	<version>1.3.2</version>
 
     <parent>
         <groupId>org.codehaus.castor</groupId>
diff --git a/schema/pom.xml b/schema/pom.xml
index 26c907c..b0be80a 100644
--- a/schema/pom.xml
+++ b/schema/pom.xml
@@ -6,6 +6,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.codehaus.castor</groupId>
     <artifactId>castor-xml-schema</artifactId>
+    <version>1.3.2</version>
 
     <parent>
         <groupId>org.codehaus.castor</groupId>
diff --git a/xml/pom.xml b/xml/pom.xml
index e6c7564..7b0ad6d 100644
--- a/xml/pom.xml
+++ b/xml/pom.xml
@@ -8,6 +8,7 @@ http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.codehaus.castor</groupId>
     <artifactId>castor-xml</artifactId>
+    <version>1.3.2</version>
     
     <parent>
         <groupId>org.codehaus.castor</groupId>
